src/hooks/use-timer-effect.hook.ts

```diff
@@ -0,0 +1,28 @@
+import { DependencyList, useEffect } from 'react';
+
+import { EmptyFn } from '../config/general';
+import { useIsAuthorisedSelector } from '../store/wallet/wallet-selectors';
+
+const effect = (callback: EmptyFn, refreshInterval: number) => {
+  callback();
+
+  let timeoutId = setTimeout(function updateData() {
```

Comment:
why `let` and not `const` ?

Reply:
![image](https://user-images.githubusercontent.com/35381314/161743339-ee1efb78-4e57-413b-857a-cc9438eb2aed.png)